lib/chef-vault/item.rb

```diff
@@ -86,18 +86,18 @@ def clients(search_or_client, action = :add)
       else
         results_returned = false
         query = Chef::Search::Query.new
-        query.search(:node, search_or_client) do |node|
+        query.search(:node, search_or_client, filter_result: { name: ["name"] }, rows: 100000) do |node|
```

Comment:
Any reason on why you limit it to 100 000 rows per page?
Also, would it be better to set it as a constant?